pages/id/[form]/settings.tsx

```diff
@@ -1,25 +1,39 @@
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
-import { requireAuthentication } from "../../../lib/auth";
-import { crudTemplates } from "../../../lib/integration/crud";
+import { requireAuthentication } from "@lib/auth";
+import { crudTemplates } from "@lib/integration/crud";
 import FormSettings from "../../../components/containers/DynamicForm/Settings";
 
 export const getServerSideProps = requireAuthentication(async (context) => {
   const formID = context && context.params && context.params.form ? context.params.form : undefined;
 
-  // get form info from db
-  const payload = {
-    method: "GET",
-    formID: formID as string,
-  };
-  const lambdaResult = await crudTemplates(payload);
-
-  if (context.locale && lambdaResult.data.records && lambdaResult.data.records.length > 0) {
-    return {
-      props: {
-        form: lambdaResult.data.records[0],
-        ...(await serverSideTranslations(context.locale, ["common", "admin-templates"])),
-      },
+  if (formID) {
+    // get form info from db
+    let formIDNumber;
+    try {
+      formIDNumber = parseInt(formID as string);
```

Comment:
I don't think this will behave as intended.  In JS parseInt() does not throw an error if it cannot parse a string to integer but instead returns NaN.  Since it returns a value and doesn't throw then the catch block will never run.